docusaurus/docusaurus.config.js

```diff
@@ -27,7 +27,14 @@ const config = {
     locales: ['en'],
   },
 
-  themes: ['@docusaurus/theme-live-codeblock'],
+  // ** WARNING! DELETE THE NEXT LINE BEFORE MERGING WITH MAIN 👇 **/
+  noIndex: true, // prevents the website from being indexed by search engines
```

Comment:
What is this here for?

Reply:
Hah, that much I understand. But it seems dangerous to rely on someone remembering to remove something. Can we use an env variable and leave it in, for example?

Reply:
[Fixed](https://github.com/strapi/documentation/pull/1698/commits/5ea854a4d0dd50ec519e0ae6ba5140d382fa7f31) ✅